target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -69,6 +69,7 @@ ramips_setup_interfaces()
 	dir-320-b1|\
 	dir-610-a1|\
 	dir-615-h1|\
+	dwr-512-b|\
```

Comment:
I modify the dts and removed the line in 02_network.
I recompiled and verified that it works as expected.